apps/client/src/common/components/input/time-input/TimeInput.module.scss

```diff
@@ -1,7 +1,7 @@
-@use "../../../../theme/v2Styles" as *;
+@use '../../../../theme/v2Styles' as *;
```

Comment:
I realise this was probably your linter, but it is generally good to try to stick to the necessary changes.